### ISSUE TITLE
ULX causing major framerate drops

### ISSUE THREAD

**Issue by Dudemanjude** [Nayruden/Ulysses#508]
Whenever I have ULX installed, it always makes my framerate go down by atleast 10 or 15 FPS. Maybe even more. Please fix this, because I want to be able to use ULX with my server.

**Comment by MrUnknownGamer** [comment]
You must have a potato computer if ULX is affecting your frame rate.

**Comment by Nayruden** [comment]
I can run ULX on my laptop under a VM and I have no drop; what are you running? No one has ever complained about this before.

**Comment by Dudemanjude** [comment]
Strange, I'm using Windows 10, and it seems like this has happened before to me.
One day, I downloaded ULX being like, this is cool! And then I noticed that my framerate was bad, but I got used to it. Now, I redownloaded GMod and now I had way better FPS. Then after I downloaded ULX again, it started going down again. I was reading about this on the forums for facepunch, and someone had the same problem. I'm using a AMD CPU and GPU, if that would do anything. The other computer with the drops was a bad laptop. It has a Intel Graphics card. Is there anything in the code that runs over and over again every second or something? It still does it if I host a server on a better laptop, with a better CPU and GPU. The game also works good normally, about 50 to 60 FPS.
